### ISSUE TITLE
Uncaught Error: socket hang up

### ISSUE THREAD

**Issue by ishaand** [atom/atom#5035]
[Enter steps to reproduce below:]

1. atom
2. ...

**Atom Version**: 0.170.0
**System**: linux 3.16.2-031602-generic
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: socket hang up

```
At events.js:85

Error: socket hang up
  at createHangUpError (_http_client.js:214:15)
  at TLSSocket.socketOnEnd (_http_client.js:292:23)
  at TLSSocket.emit (events.js:129:20)
  at _stream_readable.js:907:16
  at process._tickCallback (node.js:386:11)

```

### Commands

```
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  },
  "editor": {
    "invisibles": {}
  }
}
```

### Installed Packages

```coffee
# User
autocomplete-plus, v1.1.0
autocomplete-snippets, v0.3.2
file-icons, v1.4.5
linter, v0.10.0
minimap, v3.5.4

# Dev
No dev packages
```

**Comment by ishaand** [comment]
See https://github.com/atom/atom/issues/4974

**Comment by kevinsawicki** [comment]
Was the settings view open when this happened?

**Comment by kevinsawicki** [comment]
I believe this was fixed by https://github.com/atom/settings-view/commit/3fe0db5dc1c7db9f936b506a126d689057a24e0d which was included in the latest release, 0.172
